plugins/adrv9002.c

```diff
@@ -0,0 +1,1255 @@
+/**
+ * ADRV9002 (Navassa) Plugin
+ *
+ * Copyright (C) 2020 Analog Devices, Inc.
+ *
+ * Licensed under the GPL-2.
+ *
+ **/
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+#include <glib.h>
+#include <gtk/gtk.h>
+
+#include "../osc.h"
+#include "../osc_plugin.h"
+#include "../iio_widget.h"
+#include "../iio_utils.h"
+#include "../config.h"
+#include "dac_data_manager.h"
+
+#define THIS_DRIVER "ADRV9002"
+#define PHY_DEVICE "adrv9002-phy"
+#define DDS_DEVICE "axi-adrv9002-tx"
+#define CAP_DEVICE "axi-adrv9002-rx"
+
+#define ADRV9002_NUM_CHANNELS	2
+
+/* Max nr of widgets per channel */
+#define NUM_MAX_WIDGETS	10
+#define NUM_MAX_DDS	2
+#define NUM_MAX_ADC	2
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+const gdouble mhz_scale = 1000000.0;
+const gdouble k_scale = 1000.0;
```

Comment:
Looks like this is not used in this plugin. Could be removed.

Reply:
Yeps. It was in the beginning but now it's just a leftover...